### PR TITLE
Change team members to regression contact points

### PR DESCRIPTION
**Pull request by mm4tt**
Also add context around it.

### CODE REVIEW

File: sig-scalability/README.md [hunk 1]
```diff
@@ -78,10 +78,15 @@ Note that the links to display team membership will only work if you are a membe
 
 <!-- BEGIN CUSTOM CONTENT -->
 # SIG-Scalability Team Members
-* Janek Łukaszewicz (**[@oxddr](https://github.com/oxddr)**), Google
+
```
Comment (wojtek-t) [inline sig-scalability/README.md]: Maybe we should change the title?
To sth like scalability-regression contact-points? or sth like that?

Comment (mm4tt) [reply sig-scalability/README.md]: Done.

Comment (wojtek-t) [reply sig-scalability/README.md]: Not really yet :)

Comment (mm4tt) [reply sig-scalability/README.md]: Doh, done :)

**Pull request merged**